src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.awt.*;
```

Comment:
It seems you've imported the `java.awt.*` package, but you're not using it in your code. It's a good practice to remove unused imports to keep your code clean.